ehsm_kms_service/ehsm_kms_params.js

```diff
@@ -126,8 +127,8 @@ const ehsm_kms_params = {
     },
   },
   ExportDataKey: {
-    cmk_base64,
-    ukey_base64: {
+    keyid,
+    ukeyid: {
```

Comment:
keyid is cmk id, ukeyid is data key id. right?

Reply:
no，NAPI_ExportDataKey  use two cmk （EH_AES_GCM_128 and EH_RSA_3072)